pkg/trace/pb/stats.proto

```diff
@@ -40,6 +40,8 @@ message ClientStatsPayload {
 	// Tags specifies a set of tags obtained from the orchestrator (where applicable) using the specified containerID.
 	// This field should be left empty by the client. It only applies to some specific environment.
 	repeated string tags = 12;
+	// summaryUsesSeconds indicates when the durations in the summaries are in seconds (and not in nanoseconds).
+	bool summaryUsesSeconds = 13;
```

Comment:
Would `usingSeconds` be too ambiguous?

Reply:
> What do you think? Should we just switch everything to seconds?
> Right now, in the backend, we convert everything to seconds anyway.

In that case yes, why not? Not sure what drawbacks you mean.

Reply:
I raised the question with the stats team. And @kristinekunapuli pointed out that there is also `bucketStart` and `bucketDuration` that we would need to change.
So that's going to impact a lot of services and it makes this change more error prone. So I think it's better to just update the sketches for now. What do you think?

Reply:
I'll leave it up to you. If you want to update everything, and are unsure, you can add a feature flag `disable_stats_seconds` or similar which reverts to the old way, just in case. 100% your call.